backend/src/test/java/com/woowacourse/gongseek/comment/presentation/CommentControllerTest.java

```diff
@@ -0,0 +1,152 @@
+import com.woowacourse.gongseek.comment.presentation.dto.CommentRequest;
+import com.woowacourse.gongseek.comment.presentation.dto.CommentResponse;
+import com.woowacourse.gongseek.config.RestDocsConfig;
+import com.woowacourse.gongseek.member.presentation.dto.AuthorDto;
+import java.time.LocalDateTime;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.restdocs.AutoConfigureRestDocs;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.restdocs.payload.JsonFieldType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+
+@DisplayName("댓글 문서화")
+@AutoConfigureRestDocs
+@WebMvcTest(CommentController.class)
+@Import(RestDocsConfig.class)
+class CommentControllerTest {
+
+    @MockBean
+    private CommentService commentService;
+
+    @MockBean
+    private JwtTokenProvider jwtTokenProvider;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    void 댓글_생성_API_문서화() throws Exception {
+        CommentRequest request = new CommentRequest("content");
+
+        given(jwtTokenProvider.validateToken(any())).willReturn(true);
+        given(jwtTokenProvider.getPayload(any())).willReturn("1");
+
+        ResultActions results = mockMvc.perform(post("/api/articles/{articleId}/comments", 1)
+                .header(HttpHeaders.AUTHORIZATION, "Bearer token")
+                .content(objectMapper.writeValueAsString(request))
+                .contentType(MediaType.APPLICATION_JSON)
+                .characterEncoding("UTF-8"));
+
+        results.andExpect(status().isCreated())
+                .andDo(print())
+                .andDo(document("comment-create",
+                                requestHeaders(
+                                        headerWithName(HttpHeaders.AUTHORIZATION).description("Bearer + 토큰")
+                                ),
+                                requestFields(
+                                        fieldWithPath("content").type(JsonFieldType.STRING).description("내용")
+                                )
+                        )
+                );
+    }
+
+    @Test
+    void 댓글_조회_API_문서화() throws Exception {
+        CommentResponse commentResponse = new CommentResponse(
+                1L, "content",
+                new AuthorDto("authorName", "authorAvatarUrl"),
+                true, LocalDateTime.now()
+        );
+        given(commentService.findByArticleId(any(), any())).willReturn(List.of(commentResponse));
+
```

Comment:
```suggestion
        given(commentService.findByArticleId(any(), any())).willReturn(List.of(commentResponse1, commentResponse2));

```
2개를 find해서 API 문서화하면 Array로 담는다는 걸 명확히 알 수 있을 것 같아요!